server/src/test/java/org/opensearch/rest/BaseRestHandlerTests.java

```diff
@@ -383,4 +388,14 @@ public String getName() {
         }
     }
 
+    public void testReplaceRoutesMethod() throws Exception {
+        List<Route> routes = Arrays.asList(new RestHandler.Route(Method.GET, "/path/test"),
+            new RestHandler.Route(Method.GET, "/path2/test"));
+        List<ReplacedRoute> replacedRoutes = RestHandler.replaceRoutes(routes, "/prefix", "/deprecatedPrefix");
+
+        for(int i = 0; i < routes.size(); i++) {
+            assertEquals(replacedRoutes.get(i).getPath(), "/prefix" + routes.get(i).getPath());
```

Comment:
Why the assert is limited to path only? Should it check method and deprecated path as well?

Reply:
Done